include/openenclave/bits/attestation.h

```diff
@@ -85,7 +85,6 @@ typedef enum _oe_sgx_endorsements_fields_t
     OE_SGX_ENDORSEMENT_FIELD_CRL_PCK_CERT,
     OE_SGX_ENDORSEMENT_FIELD_CRL_PCK_PROC_CA,
     OE_SGX_ENDORSEMENT_FIELD_CRL_ISSUER_CHAIN_PCK_CERT,
-    OE_SGX_ENDORSEMENT_FIELD_CRL_ISSUER_CHAIN_PCK_PROC_CA,
```

Comment:
Since this is a public header, I suggest adding a note in the release notes for this change.

Reply:
Thanks. I checked with @radhikaj  and she feels it's fine to add that in the CHANGE.log as I'm fixing a bug.

Reply:
The API that uses these fields are in endorsements.h which is not public. So we are ok.

---
In reply to: [424736127](https://github.com/openenclave/openenclave/pull/3018#discussion_r424736127) [](ancestors = 424736127)

Reply:
At some point, we should move SGX-specific stuff out of this non-SGX-specific header, but not in this PR.